scripts/optimizer.sh

```diff
@@ -0,0 +1,16 @@
+:
```

Comment:
What does this mean?

Reply:
It means something like "use the shell that calls this script". I'll change it to bash.